tdr_preingest.tf

```diff
@@ -2,6 +2,8 @@ locals {
   tdr_preingest_name              = "${local.environment}-dr2-preingest-tdr"
   tdr_aggregator_name             = "${local.tdr_preingest_name}-aggregator"
   tdr_aggregator_queue_arn        = "arn:aws:sqs:eu-west-2:${data.aws_caller_identity.current.account_id}:${local.tdr_aggregator_name}"
+  tdr_aggregator_batching_window  = 300
```

Comment:
Maybe "in_secs" could be appended to this name?

Reply:
I'll go with `tdr_aggregator_batching_window_seconds` to match some of the other ones.